lib/rspec/rails/view_rendering.rb

```diff
@@ -39,18 +39,50 @@ def render_views?
         self.class.render_views? || !controller.class.respond_to?(:view_paths)
       end
 
-      # Delegates find_all to the submitted path set and then returns templates
-      # with modified source
-      #
-      # @private
-      class EmptyTemplateResolver < ::ActionView::FileSystemResolver
+      class EmptyTemplateResolverFactory
```

Comment:
Actually, we could move `EmptyTemplateFileSystemResolver` and `EmptyTemplateResolverDecorator` into this `EmptyTemplateResolver` module (class), since they’re already supposed to be private and are only used by this factory class.